settings.js

```diff
@@ -4,7 +4,7 @@ const packageJSON = require('./package.json');
 const defaultBuildDestination = path.resolve("./build");
 
 module.exports = {
-  buildDestination: process.env.BUILD_PATH || defaultBuildDestination,
+  buildDestination: "C:\\Users\\Ahmed\\Documents\\Qlik\\Sense\\Extensions\\KPI",
```

Comment:
This is the problem. Build-folder has changed which means that the build folder cannot be persisted to next job.